.github/styles/vocab.txt

```diff
@@ -24,6 +24,7 @@ GMail
 Grav
 Gravatar
 href
+html
```

Comment:
```suggestion
html
```

This should not be lower case - it should be capitalised.

Reply:
Thanks. I copied it from the Vale output which had it as lowercase. We'll have to figure out how we can keep the Vale style files in sync at some point. I'll put a ticket here so we don't lose track of that.